common/lib/xmodule/xmodule/js/src/sequence/display.js

```diff
@@ -30,9 +30,6 @@
             this.toggleArrows = function() {
                 return Sequence.prototype.toggleArrows.apply(self);
             };
-            this.updateProgress = function() {
```

Comment:
Don't trust me-- you should do a search also to make sure I didn't miss a reference. :)